testsuite/features/min_ubuntu_salt_install_package_and_patch.feature

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2019 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Install and upgrade package on the Ubuntu minion via Salt through the UI
+
+@ubuntu_minion
+  Scenario: Pre-requisite: install virgo-dummy-1.0 packages
+    When I enable repository "Devel_Uyuni_BuildRepo" on this "ubuntu-minion"
+    And I run "apt update" on "ubuntu-minion"
+    And I remove package "andromeda-dummy" from this "ubuntu-minion"
+    And I install package "virgo-dummy=1.0" on this "ubuntu-minion"
+    And I am on the Systems overview page of this "ubuntu-minion"
+    And I follow "Software" in the content area
+    And I click on "Update Package List"
+    And I follow "Events" in the content area
+    And I wait until I do not see "Package List Refresh scheduled by admin" text, refreshing the page
+
+@ubuntu_minion
+  Scenario: Install a package on the minion
+    Given I am on the Systems overview page of this "ubuntu-minion"
+    And I follow "Software" in the content area
+    And I follow "Install"
+    And I check "andromeda-dummy" in the list
+    And I click on "Install Selected Packages"
+    And I click on "Confirm"
+    And I should see a "1 package install has been scheduled for" text
+    When I wait until event "Package Install/Upgrade scheduled by admin" is completed
+    Then Deb package "andromeda-dummy" with version "2.0" should be installed on "ubuntu-minion"
+
+@ubuntu_minion
+  Scenario: Update a package on the minion
+    Given I am on the Systems overview page of this "ubuntu-minion"
+    And I follow "Software" in the content area
+    And I follow "Upgrade" in the content area
+    And I check "virgo-dummy-2.0-X" in the list
```

Comment:
I don't have a running server atm so I cannot confirm it but I think the '-X' suffixes are stripped in the list. Need to double-check this.

Reply:
looks strange to me, at least

Reply:
I checked again and '-X' suffixes actually exist in the list indeed. So this step is correct after all.

@Bischoff, yes, it's exclusive to Debian, because 'release' number is optional in deb package names. And '-X' suffix is a placeholder that `repo-sync` adds because it's mandatory on our DB.